src/rubintv/handlers/external/endpoints.py

```diff
@@ -137,25 +138,35 @@ async def get_recent_table(request: web.Request) -> web.Response:
             bucket = request.config_dict["rubintv/gcs_bucket"]
             events = get_most_recent_day_events(bucket, camera)
             channels = camera.channels
-            grid_columns = build_grid_columns_css(channels)
+            the_date = events[0].cleanDate()
+
+            metadata_json = "{}"
+            metadata_url = get_metadata_url(bucket.name, camera.slug, the_date)
+            metadata_res = requests.get(metadata_url)
+            if metadata_res.status_code == 200:
+                metadata_json = metadata_res.text
+
             page = get_formatted_page(
                 "cameras/camera.jinja",
                 title=title,
                 camera=camera,
                 channels=channels,
-                date=events[0].cleanDate(),
+                date=the_date,
                 events=events,
-                grid_columns=grid_columns,
+                metadata=metadata_json,
             )
+
     logger.info("get_recent_table", duration=timer.seconds)
     return web.Response(text=page, content_type="text/html")
 
 
-def build_grid_columns_css(channels: Dict[str, Channel]) -> str:
-    grid_columns = "1fr"
-    for channel in channels:
-        grid_columns += " 40px"
-    return grid_columns
+def get_metadata_url(bucket_name: str, camera_slug: str, date_str: str) -> str:
+    #  reformat the date string from YYYY-m-d to YYYYmmdd
+    date_str = "".join([f"{int(x):02}" for x in date_str.split("-")])
+
+    url = f"https://storage.googleapis.com/{bucket_name}/"
```

Comment:
You might think about using `urllib` and `urljoin` to build up your URLs.  It might be a little more maintainable.

Reply:
I think this is a good suggestion in principle. However, as we discussed yesterday, I think we're going to move the whole app to using the GCS API instead of URLs at all, so I think this is OK to leave for now, and we'll tidy the whole thing up very soon.